addons/Group_Chat_Button/test/ModelValidationTests.js

```diff
@@ -0,0 +1,57 @@
+TestCase("[Group Chat Button] Model validation", {
+    setUp: function () {
+        this.presenter = AddonCross_Lesson_create();
+    },
+
+    'test given correct model when validating then return correct configuration': function () {
```

Comment:
poprosiłbym o przetestowanie także, czy is visible na false dobrze działa. We wszystkich testach masz `'Is Visible': "True"`